src/main/kotlin/br/com/creditas/projethom/controller/TeamController.kt

```diff
@@ -38,4 +44,16 @@ class TeamController(
         @RequestBody @Valid newTeamRequest: TeamRequest
     ): TeamResponse = teamService.registerTeam(newTeamRequest)
 
+    @PutMapping
+    fun putTeam(
+        @RequestBody @Valid updateTeamRequest: UpdateTeamRequest
+    ): TeamResponse = teamService.updateTeam(updateTeamRequest)
+
+    @DeleteMapping("/{id}")
```

Comment:
Seria legal validar se o id existe antes de tentar excluir

Reply:
Acho que ataco isso quando for mexer melhor nas exceptions. Ai já deixo todas exceptions padronizadas